base/i18n/time_formatting.cc

```diff
@@ -260,7 +260,7 @@ std::u16string DateIntervalFormat(const Time& begin_time,
   UErrorCode status = U_ZERO_ERROR;
 
   std::unique_ptr<icu::DateIntervalFormat> formatter(
-#if defined(USE_HACKY_COBALT_CHANGES)
```

Comment:
@andrewsavage1 Did you see ICU related bug after this commit: https://github.com/youtube/cobalt/pull/3435? Because using the wrong macro ```USE_HACKY_COBALT_CHANGES``` basically means the code not going through this line, then do we still need this customization?

Reply:
I think we should just get rid of it